assets/javascripts/initializers/ai-bot-replies.js

```diff
@@ -145,6 +146,28 @@ function initializeShareButton(api) {
   const modal = api.container.lookup("service:modal");
 }
 
+function initializeShareTopicButton(api) {
+  const siteSettings = api.container.lookup("site-settings:main");
+  api.registerTopicFooterButton({
+    id: "share-ai-conversation",
+    icon: "share",
+    label: "discourse_ai.ai_bot.share_ai_conversation.name",
+    title: "discourse_ai.ai_bot.share_ai_conversation.title",
+    action() {
+      const modal = api.container.lookup("service:modal");
```

Comment:
Should do this lookup in the same place as `siteSettings` above, otherwise you have to look it up every time you click the button

Reply:
true will change